Source/Core/TerrainMesh.js

```diff
@@ -45,7 +44,6 @@ function TerrainMesh(
   vertexStride,
   orientedBoundingBox,
   encoding,
```

Comment:
`TerrainMesh` doesn't need exaggeration any more. We send the un-exaggerated positions to the GPU and scale dynamically. In JS we would call `TerrainEncoding.getExaggeratedPosition`